Dockerfile

```diff
@@ -34,6 +34,11 @@ RUN if [ "${HTCONDORCERN}" -eq 1 ]; then \
   apt-get -y remove gnupg2 wget alien; \
 fi;
 
+ARG SLURM
```

Comment:
Now that we have the third backend, perhaps we should start using backend strings that will also address https://github.com/reanahub/reana-job-controller/issues/170?

Reply:
I did so in #178